swebench/harness/constants.py

```diff
@@ -372,13 +372,17 @@
             "pytest-mock==3.11.1", "pytest-openfiles==0.5.0", "pytest-remotedata==0.4.0", "pytest-xdist==3.3.1",
             "pytest==7.4.0", "PyYAML==6.0.1", "setuptools==68.0.0", "sortedcontainers==2.4.0", "tomli==2.0.1",
```

Comment:
Oh interesting, I tried to do this enforcement originally in the list of pip packages to install, but because the `pip install -e .` uses the `pyproject.toml` file, I'm assuming it doesn't get enforced?

But this looks great, as long as it's working that's good, thanks so much for catching the install fails + proposing the fix 😄